### PR TITLE
Events

### PR DESCRIPTION
**Pull request by PrinceBadru**
## Math magicians : Event.
- Create a new directory called logic inside your project
- Copy the files calculate.js and operate.js into your logic/ directory
- Install [big.js](https://www.npmjs.com/package/big.js) using NPM
- Analyze the files calculate.js and operate.js, be sure you understand what they do
- Import calculate.js in your Calculator.js component

### CODE REVIEW

**Review by V-Blaze — CHANGES_REQUESTED** [review]
Hi @PrinceBadru 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

dcentl8-9b96e129-9725-401b-ac34-2feb990202a0

## Invalid Code Review Request

## You have submitted a project, that doesn't follow the correct Git flow. You are supposed to create a pull request to merge into the `development` branch.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by PrinceBadru** [comment]
hi @V-Blaze  so should I delete this pull request and restart everything?

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,19 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
+import Calculator from './components/Calculator';
+import calculate from './logic/calculate';
```
Comment (fobadara) [inline src/App.js]: [REQUIRED] You are expected to import `calculate` into your calculator component. If you notice it your way complicates things and involves a lot of passing props down. When the clicks could have simply been handled in the component. Kindly fix this.

File: src/components/Calculator.js
Comment (fobadara) [inline src/components/Calculator.js:1]: [REQUIRED] When I try to divide numbers I get an error:
image
Kindly fix this.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @PrinceBadru, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

REDO

## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code
✅ Good first attempt
✅ Good implementation of the principle of separating concerns.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @PrinceBadru, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alusp — COMMENTED** [review]
Hi @PrinceBadru, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
congrats
## Highlights

- [ ] Handle events in a React app. 👍
- [ ] Use Professional Readme file 👍
- [ ] Descriptive PR Summary used 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**